### ISSUE TITLE
False warning : parameterised class parameter without a default value

### ISSUE THREAD

**Issue by nvogel** [rodjek/puppet-lint#137]
Hi,

I've the following warning but there's no parameterised class in the manifest.

**WARNING: parameterised class parameter without a default value on line 4**

``` puppet
# = Class : myclass
class myclass {

    if ( $::lsbdistcodename == 'squeeze' ) {
      #TODO
    }
}
```

With puppet-lint (0.2.0)
